### ISSUE TITLE
Add browserify support

### ISSUE THREAD

**Issue by wtfaremyinitials** [PrismarineJS/prismarine-server#16]
Could be fun to have people running servers and clients in the browser, players could ultimately connect with WebRTC.

**Comment by dcbartlett** [comment]
if we do this, i think it should just be a vanilla based server, very limited in its capabilities.  The host may not have a beefy enough machine to run a client and the server at the same time if too many people connect.

**Comment by roblabla** [comment]
The notchian minecraft client already runs a local server.

**Comment by dcbartlett** [comment]
right, but thats on the pc, and most people don't "share" that one, they use real servers.  WebRTC is going to tax the system much more heavily than a pre-built java runtime.  At least thats just my opinion.
